### PR TITLE
docs(database): Be more explicit about operators/functions in CDS files

### PR DESCRIPTION
**Pull request by bugwelle**
The functions/operators are only supported in the new SQL database service, but not by the compiler in CDS files. Let's be more clear about that.

### CODE REVIEW

**Review by patricebender — APPROVED** [review]
Thanks :)

File: guides/databases-sqlite.md [hunk 1]
```diff
@@ -419,6 +420,7 @@ A specified set of standard functions is now supported in a **database-agnostic*
 * `year` `month`, `day`, `hour`, `minute`, `second` — return parts of a datetime
 
 > <sup>1</sup> Argument `n` is optional.
+> These functions are only supported at runtime via the database service, but in CDS files.
```
Comment (renejeglinsky) [inline guides/databases-sqlite.md:423]: Should it say "not"?
```suggestion
> These functions are only supported at runtime using the database service, but not in CDS files.
```

Comment (bugwelle) [reply guides/databases-sqlite.md]: Yes, thanks! 🤦‍♂️

File: guides/databases-sqlite.md [hunk 2]
```diff
@@ -419,6 +420,7 @@ A specified set of standard functions is now supported in a **database-agnostic*
 * `year` `month`, `day`, `hour`, `minute`, `second` — return parts of a datetime
 
 > <sup>1</sup> Argument `n` is optional.
+> These functions are only supported at runtime using the database service, but not in CDS files.
```
Comment (johannes-vogel) [inline guides/databases-sqlite.md:423]:
```suggestion
> These functions are only supported within runtime queries, but not in CDS files.
```

**Review by johannes-vogel — APPROVED** [review]
lgtm

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**